src/domain-registry-essentials/architecture-overview.md

```diff
@@ -1,123 +1,135 @@
+`Registry` is the most essential smart contract in CNS. This is the contract that defines ownership rules, how domains are minted, provides [ERC-721 token metadata](https://docs.openzeppelin.com/contracts/2.x/api/token/erc721#IERC721Metadata), and stores a metadata-enriched list of all domains.
 
-Registry stores the following data about each domain:
+`Registry` stores:
 
-* Owner address
-* Approved operator address
-* Resolver address
-* Domain name
+- Owner address
+- Approved operator address
+- Resolver address
+- Domain name
 
 {% hint style="info" %}
-As domains are identified by a namehash, we also store a domain name, apart from the other data. As it's impossible to calculate a domain name from its identifier, it allows users, that know only a namehash, lookup a real domain name.
+Since domains are identified by a namehash we also explicitly store the domain's name. This allows users who only know a namehash to look up a record by its name.
 {% endhint %}
 
-Registry smart contract implements a set of methods allowing to mint new domains, subdomains, and manage ownership.
+`Registry`'s smart contract includes a set of methods for minting new domains, creating new subdomains, and managing ownership.
 
-Accounts that are allowed to mint second-level domains \(like 'alice.crypto'\) are called whitelisted minters. Whitelisted minters are limited to only minting new domains. They can't control domain ownership \(e.g. approve or transfer a domain to another owner\) and they can't control domain records. Whitelisted minters are operated by Unstoppable Domains.
+Accounts that are allowed to mint second-level domains (e.g.: `alice.crypto`) are called whitelisted minters. Whitelisted minters are only permitted to mint new domains. They can't control domain ownership (e.g. approve or transfer a domain to another owner) and they can't change domain records. Whitelisted minters are operated by Unstoppable Domains.
 
-Registry smart contract doesn't have an admin, which means that no entity can transfer or manage user domains without their permission.
+`Registry`'s smart contract was designed without an admin. This means that no entity can manage or transfer a user's domains without their permission — even Unstoppable Domains.
 
-A domain owner may perform the following operations with their domains:
+Domain owners can:
 
-* Transfer ownership
-* Set a new resolver
-* Mint a new subdomain
-* Burn a domain
+- Transfer domain ownership
+- Set a new resolver
+- Mint a new subdomain
+- Burn a domain
 
-Users can set one approved address per owned domain and many operators, that can manage their domains. For more details, see [Managing domain ownership](../managing-domains/managing-domain-ownership.md).
+Domain owners can set one _Approved address_ per domain and many _Operator_ addresses. These Operators can manage a domain on a user's behalf. For more details, see [Managing Domain Ownership](../managing-domains/managing-domain-ownership.md).
```

Comment:
> These Operators can manage a domain on a user's behalf

Approved addresses can do that as well. I'd try to generalize these two roles without using the words "approved" or "operator"